packages/create-next-app/create-app.ts

```diff
@@ -93,6 +94,15 @@ export async function createApp({
   }
 
   const root = path.resolve(appPath)
+
+  if (!(await isWriteable(path.dirname(root)))) {
+    console.error('The operation was rejected by your operating system.')
+    console.error(
+      'It is likely you do not have write permissions for this folder as the current user'
```

Comment:
Not sure we want to mention "as the current user" this might inadvertently suggest they elevate privileges to admin/root since it mentions lacking permission 🤔